### ISSUE TITLE
"********* listening on port 6007" message in console everytime scene is run

### ISSUE THREAD

**Issue by freemanfromgodotengine** [godotengine/godot#3563]
Everytime scene is run (debug) has a console spit that reads:
```
********* listening on port 6007
OK! 72
has connection!
has connection!
CLOSING CONNECTION
```

**Comment by Algrin6** [comment]
I think it's not a bug. Those messages just note you about a successful connection between the editor and your game, which is used to debug your game.

**Comment by neikeq** [comment]
I find these messages spammy, specially when implementing an HTTP Server whose clients are mostly using [request](https://github.com/request/request) (which creates a new connection for each request).

**Comment by djrm** [comment]
tracker here #5031, may be solved by this #5283

**Comment by djrm** [comment]
closed by #5283
